diff --git a/node_modules/localtunnel/lib/HeaderHostTransformer.js b/node_modules/localtunnel/lib/HeaderHostTransformer.js
index 18813ef..c588ffd 100644
--- a/node_modules/localtunnel/lib/HeaderHostTransformer.js
+++ b/node_modules/localtunnel/lib/HeaderHostTransformer.js
@@ -1,4 +1,4 @@
-const { Transform } = require('stream');
+const { Transform } = require('readable-stream');
 
 class HeaderHostTransformer extends Transform {
   constructor(opts = {}) {
